save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -72,4 +73,19 @@ class VulnerabilityController(
     }.map {
         ResponseEntity.ok("Vulnerability was successfully saved")
     }
+
+    @PostMapping("/save-all-projects")
```

Comment:
It seems to be a little bit complicated endpoint - too many characters

Reply:
Fixed